header.go

```diff
@@ -1060,6 +1187,8 @@ func (h *RequestHeader) setSpecialHeader(key, value []byte) bool {
 // the Content-Type, Content-Length, Connection, Server, Set-Cookie,
 // Transfer-Encoding and Date headers can only be set once and will
 // overwrite the previous value.
+//
+// If the header is in Trailer, it will send after the chunked response body.
```

Comment:
```suggestion
// If the header is set as a Trailer, it will be send after the chunked response body.
```
Same for the other comments below.